www-client/netsurf/netsurf-3.8.recipe

```diff
@@ -24,6 +23,8 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libcss$secondaryArchSuffix
```

Comment:
Does it? In the previous versions these were statically linked.

Also, were the libraries all updated already? Because if you update the NetSurf frontend but keep the old libs, we don't really get any improvement to web standards support so there is little point in updating.

Reply:
@pulkomandy - Without it, it breaks the final package build process. The update provides page layout, stability and security improvements. I think the update is good moreso in low memory overhead footprint for <=1GB-2GB laptops which get easily consumed by the larger Webkit browsers.

Reply:
Of course the update is good. But we need to do it with all the libraries (libcss, libhubbub, libnspsl, libnsbmp, libnsgif, ...) otherwise we only get 10% of the work from the NetSurf team. Did you check all of them? Are they all up to date?

Reply:
I've run NetSurf 3.8 for two hours (even reviewing Youtube content (no video, no Javascript enabled)...no crashes yet and <90 MB memory usage. LibCSS is current (0.8.0) and everything was current enough for the browser to compile and work properly.

Reply:
I'll update libnspsl in #3130.

Reply:
Bumped all supporting libs.

Reply:
Everything looks good so far. @fbrosson reviewing any needed polishing.